### DISCUSSION TITLE
Lesson 3 (FundMe): why is withdraw() payable?

### DISCUSSION THREAD

**Discussion by xavierdmello** [smartcontractkit/full-blockchain-solidity-course-py#1453] [Q&A] [answered]
I've tried it both ways, and it doesn't seem to matter if it's payable or not. Just curious why withdraw was payable in the tutorial. Was it just an oversight?

**Comment by cromewar** [comment, marked as answer]
Hello @xavierdmello 
The reason si because you are transferring Ethereum to the msg.sender.
![image](https://user-images.githubusercontent.com/20199180/166150631-d206dc61-c7f5-4372-b6cc-99c1d424cd65.png)
As you are transferring the balance of the contract, the function is payable as you are "paying" to the wallet which si doing the withdraw.

**Reply by xavierdmello** [reply]
Thank you. Since the user never sends any ether to withdraw(), would it still need to be payable? Or is it just good practice etc.

**Reply by cromewar** [reply]
Yeah, the one which sends the payment is the contract so the address should be payable and in consequence de function as well.
